docs/data-model.asciidoc

```diff
@@ -457,6 +457,10 @@ For example, when `transaction.name` has a lot of unique values and reaches the
 of unique transaction groups tracked, any transactions with new `transaction.name` will be aggregated under
 `transaction.name`: `_other`.
 
+Another example is that `transaction.name` contains just a few unique values and stays within the transaction groups limit,
+but `service.name` has high cardinality and reaches the max services limit. Any transactions with new `service.name` will
+be aggregated under `service.name`: `_other` and `transaction.name`: `_other`.
```

Comment:
I was going for something different: the transaction name and the service name have low cardinality but there a service has many different instances (is deployed on a lot of hosts).


```suggestion
Another example of how the transaction group limit may be reached is if `transaction.name` contains just a few unique values,
but the service is deployed on a lot of different hosts. As `host.name` is part of the aggregation key for transaction metrics, the max transaction group limit is reached for a service that has 100 instances and 40 different transaction names when connected to an APM Server with 8GB of RAM . Once this limit is reached, any new combinations of transaction.name and host.name for that service will be aggregated under `transaction.name`: `_other`.
```

If you want to apply the suggestion verbatim, you'll probably need to adjust the line lengths.